pkg/hoist/hoist_launchable.go

```diff
@@ -259,18 +264,18 @@ func (hl *Launchable) Executables(
 			}
 
 			p2ExecArgs := p2exec.P2ExecArgs{
-				Command:          []string{filepath.Join(serviceDir, service.Name())},
+				Command:          []string{filepath.Join(hl.InstallDir(), relativePath)},
 				User:             hl.RunAs,
 				EnvDirs:          []string{hl.PodEnvDir, hl.EnvDir()},
-				ExtraEnv:         map[string]string{launch.EntryPointEnvVar: service.Name()},
+				ExtraEnv:         map[string]string{launch.EntryPointEnvVar: entryPointName},
```

Comment:
hmmmm that's interesting, it's probably better to keep the slashes. then it's just the runit service name where the double underscores happen. good suggestion will fix